webapp/packages/plugin-data-viewer/src/ValuePanelPresentation/TextValue/getTypeExtension.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+
+import { HTML_EDITOR, JSON_EDITOR, XML_EDITOR } from '@cloudbeaver/plugin-codemirror6';
+
+export function getTypeExtension(type: string) {
+  switch (type) {
+    case 'application/json':
+      return JSON_EDITOR();
+    case 'text/html':
+      return HTML_EDITOR();
+    case 'text/xml':
+      return XML_EDITOR();
+    default:
+      return;
+  }
+}
```

Comment:
just notice that languages imported synchronously and will be included in the bundle that uses this function. Please leave a comment that this import is heavy.